package.json

```diff
@@ -78,10 +78,10 @@
     "core-js": "^3.1.4",
     "debug": "^4.3.2",
     "lodash": "^4.17.10",
-    "moment": "^2.22.2",
+    "moment": "^2.30.1",
```

Comment:
I can't see where you handle date/time that will require you to upgrade the package.